.github/workflows/build-wheels-windows-dispatch_new.yml

```diff
@@ -0,0 +1,33 @@
+name: windows-dispatch
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build-python-wheels:
+    name: Build Python Wheels for windows-latest
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.8.10']
```

Comment:
Is this version motivated by the minimal ESP-IDF requirement or the Windows installer?

Windows installer used to deploy 3.8 for years. However, it has recently switched to 3.11.

Can you check please if the wheels (e.g. cryptography) built for 3.8 are usable for 3.11?

Reply:
This was motived by minimal ESP-IDF requirement.

There is only Python 3.6 version for Windows so I tested it with this for Python 3.11 and it was successfully installed.

Reply:
But now you've switched to 3.11. Why?

Reply:
Thank you for personal clarification, switched back.